php/commands/site.php

```diff
@@ -698,6 +698,60 @@ public function create_env($args, $assoc_args) {
    }
 
    /**
+    * Merge a Multidev Environment into Dev Environment
+    *
+    * ## OPTIONS
+    *
+    * [--site=<site>]
+    * : Site to use
+    *
+    * [--env=<env>]
+    * : Name of multidev to environment to merge into Dev
+    *
+    * @subcommand merge-to-dev
+    */
+    public function merge_to_dev($args, $assoc_args) {
+      $site = SiteFactory::instance(Input::site($assoc_args));
+      $site->environmentsCollection->fetch();
+
+      $multidev_ids = array_map(function($env) { return $env->id; }, $site->environmentsCollection->multidev());
+      $multidev_id = Input::env($assoc_args, 'env', "Multidev environment to merge into Dev Environment", $multidev_ids);
+      $environment = $site->environmentsCollection->get($multidev_id);
+
+      $workflow = $environment->mergeToMaster();
```

Comment:
This is again another area I would prefer to keep concrete and matching the platform's distinction between "merge multidev environment into dev environment" and "merge dev environment into multidev environment". These are operating on environments and not git branches (despite the conflation in the platform api) and may be doing other things than just merging the underlying git branches. So I'm reluctant to try to wrap the two workflows within a single method.

In other words, these are two distinct actions as part of the Pantheon Workflow, so they deserve separate methods.